src/features/goblins/hud/components/BackButton.tsx

```diff
@@ -14,27 +14,34 @@ export const BackButton = () => {
     <>
       <div className="fixed top-2 left-2 z-50">
         <OuterPanel>
-          <div className="flex justify-center p-1">
+          <div className="flex justify-center p-0.5">
             <Button
               className="bg-brown-200 active:bg-brown-200 w-full"
               onClick={() => setShowModal(true)}
             >
-              <img
-                src={arrowLeft}
-                alt="back-arrow"
+              <div
+                className="flex items-center mx-1"
```

Comment:
You should also be able to use the `flex-wrap: nowrap` to stop this from wrapping ever.